src/TimeSlotGroup.js

```diff
@@ -24,7 +24,9 @@ export default class TimeSlotGroup extends Component {
             slotProps['data-time'] = value
             slotProps['data-resource-id'] = resource
             slotProps['data-timeslot-id'] = groupId * group.length + idx
-            slotProps['aria-label'] = value && value.toLocaleTimeString()
+            slotProps['aria-label'] = slotProps['aria-label']
+              ? slotProps['aria-label']
+              : value && value.toLocaleTimeString()
```

Comment:
I think we should just get rid of this and have the consumer set the aria-label slot prop